doc/source/notebooks/tips_and_tricks.ipynb

```diff
@@ -1,8 +1,18 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "How GPflow relates to TensorFlow: Tips & Tricks \n",
+    "--\n",
+    "\n",
+    "GPflow is built on top of TensorFlow, so it is useful to have some understanding of how TensorFlow works. Particularly its two-stage concept of first building a static compute graph and then executing it for specific input values passed in leads to a bunch of \"gotchas\". This notebook aims to help with the most common issues."
```

Comment:
weird sentence.
I guess you mean : understanding this leads to "gotchas" (you miss the 'understanding')